### ISSUE TITLE
Periodic events in the calendar showed ligh-colour

### ISSUE THREAD

**Issue by jgato** [jobisoft/EAS-4-TbSync#120]
When I receive a periodic event, I have to answer if attend/maybe/no. This is ok, But. even if I say attending to a periodic event, this will show in the calendar as if I had not taken any decision yet:

![image](https://user-images.githubusercontent.com/3435696/82795183-ecd48600-9e73-11ea-92fc-2e2403535cd3.png)

In the picture, the first event has been accepted. And it is ok, painted in blue. This is not a periodic event. It is an event that happens only once.
The second event, the same day, it is shown in light-blug. This is a periodic event that I have said as attending. But still painted in light-blue colour.
![image](https://user-images.githubusercontent.com/3435696/82795322-20171500-9e74-11ea-9d6f-6d56f6262823.png)

## Your environment

TbSync version: 2.12
EAS-4-TbSync version: 1.14
Thunderbird version: 68.7

[ ] Yes, I have installed the latest available beta version from 
https://tbsync.jobisoft.de
and my issue is not yet fixed, I can still reproduce it.

## Expected behavior
When I accept attending one periodic event, this is shown as a regular-color.

## Actual behavior
When I accept attending one periodic event, this is shown as a light-clour (waiting for taken a decisoin)

## Steps to reproduce
...

**Comment by KIAOman2020** [comment]
I have similar behavior but not only with periodic events, if I accept any event it still shows light-blue with note "You haven't yet responded to this invitation". However if I'm the organizer it shows correct blue color .


Tbsync Version
2.16

TB version
78.1.1

**Comment by jgato** [comment]
any news about that?

**Comment by mschwerhoff** [comment]
Potentially a duplicate of #138.

**Comment by jobisoft** [comment]
The project has moved on to a total rewrite, the new official and only supported version on GitHub is v5:

https://github.com/jobisoft/TbSync/releases
https://github.com/jobisoft/EAS-4-TbSync/releases

If you can reproduce the issue on v5, please create a new issue with some debug info. Thank you for your support.
